docs/development/frontend.rst

```diff
@@ -17,6 +17,37 @@ however, you can trigger a manual build of them by installing all of the
 dependencies using ``npm install`` and then running ``gulp dist``.
 
 
+Tests
+-----
+
+The JavaScript codebase includes tests that can be ran via
+``make static_tests`` which is also ran with the ``make tests`` target.
+Both targets will run the tests in the Docker enviroment but they
+may also be ran locally using ``npm run test``.
```

Comment:
It could be useful here to note the version of node/npm required to run the tests locally.

Reply:
Good point, I don't think there's any mention to installing nodejs in the docs, I'll add that. 👍

Reply:
This has been addressed.